sale_order_margin_percent/models/sale_order_margin.py

```diff
@@ -12,5 +12,6 @@ class SaleOrder(models.Model):
 
     @api.depends('margin', 'amount_untaxed')
     def _compute_percent(self):
-        if self.margin and self.amount_untaxed:
-            self.percent = (self.margin / self.amount_untaxed) * 100
+        for order in self:
+            if order.margin and order.amount_untaxed:
```

Comment:
It is not really part of this PR which is an actual bugfix waiting to be merged for quite some time although there is always space to improve. Not sure how it works here.